packages/core/src/components/Carousel/Carousel.js

```diff
@@ -1,48 +1,77 @@
+import 'mutationobserver-shim';
+
 import PropTypes from 'prop-types';
 import React, { useState, useEffect } from 'react';
 import cx from 'classnames';
 import NukaCarousel from 'nuka-carousel';
-import shortid from 'short-id';
+import nanoid from 'nanoid';
 import Icon from '../Icon/Icon';
-import SectionHeader from '../Type/SectionHeader/SectionHeader';
+import BtnContainer from '../BtnContainer/BtnContainer';
+import ScreenReadable from '../ScreenReadable/ScreenReadable';
 
 import css from './Carousel.css';
 
 const Carousel = props => {
+  const id = nanoid();
   const [slideIndex, setSlideIndex] = useState(0);
 
-  const renderDesktopControls = () => {
+  const renderTopRightControls = () => {
+    if (!props.useTopRightControls) return null;
     return (
       <div className={css.controls}>
-        <SectionHeader title={props.title} level={2} className={css.title} />
+        <h2 className={css.title}>{props.title}</h2>
         {props.slidesToShow < props.children.length && (
           <div className={css.buttons}>
-            <button
-              className={cx(css.button)}
-              disabled={slideIndex === 0}
+            <BtnContainer
               onClick={() => setSlideIndex(slideIndex - 1)}
+              className={css.button}
+              disabled={slideIndex === 0}
             >
-              <Icon className={cx(css.chevron, css.prevIcon)} name="chevron-right" />
-            </button>
-            <button
-              className={cx(css.button)}
-              disabled={slideIndex === props.children.length - props.slidesToShow}
+              <Icon className={cx(css.chevron, css.prevIcon)} name="chevron" />
+              <ScreenReadable>{props.accessibilityPrevLabel}</ScreenReadable>
+            </BtnContainer>
+            <BtnContainer
               onClick={() => setSlideIndex(slideIndex + 1)}
+              className={css.button}
+              disabled={slideIndex === props.children.length - props.slidesToShow}
             >
-              <Icon className={cx(css.chevron, css.nextIcon)} name="chevron-right" />
-            </button>
+              <Icon className={cx(css.chevron, css.nextIcon)} name="chevron" />
+              <ScreenReadable>{props.accessibilityNextLabel}</ScreenReadable>
+            </BtnContainer>
           </div>
         )}
       </div>
     );
   };
 
+  const renderCenterLeftcontrols = () => {
+    if (!props.useCenterControls || (!props.infinite && slideIndex === 0)) return null;
```

Comment:
Again I think it reads better to move the condition `!props.useCenterControls` outside of the function and only call when necessary:
```jsx
{props.useCenterControls && renderCenterLeftControls()}
```